app/scripts/controllers/console.js

```diff
@@ -115,6 +125,11 @@ angular.module('console', ['sql'])
             $scope.error.hide = false;
             $scope.renderTable = false;
             $scope.error.message = sqlQuery.error.message;
+
+            $scope.error.error_trace = sqlQuery.error.error_trace;
+            if(!$scope.showErrorTrace){
+              $scope.toggleOptions();
```

Comment:
we should show the hint to the `error_trace` option once
you can store it in the `localStorage`